tests/plugins/test_generate.py

```diff
@@ -0,0 +1,46 @@
+from qenerate.core.code_command import plugins
+import pytest
+from qenerate.plugins.pydantic_v1.plugin import AnonymousQueryError, InvalidQueryError
+
+
+@pytest.mark.parametrize(
+    "query",
+    [
+        # simple query
+        "saas_file_simple",
+        # interface query
+        "complex_inline_fragments_and_collisions",
+        # interface humongous query
+        "saas_humongous",
+        # difficult attribute name query
```

Comment:
nit: those comments feel redundant. Ideally, the filename already describes the case properly